electron/game_config.ts

```diff
@@ -252,19 +246,36 @@ class GameConfigV0 extends GameConfig {
       showMangohud,
       targetExe,
       useGameMode,
-      winePrefix: winePrefix || `${userHome}/.wine`,
-      wineCrossoverBottle,
-      wineVersion,
       useSteamRuntime,
-      ...settings[this.appName],
-      language: ''
+      language: '' // we want to fallback to '' always here, fallback lang for games should be ''
     } as GameSettings
 
-    if (gameWinePrefix) {
-      settingsToReturn.winePrefix = gameWinePrefix.replace('~', userHome)
+    let gameSettings = {} as GameSettings
+
+    if (existsSync(this.path)) {
+      // read game's settings
+      const settings = JSON.parse(readFileSync(this.path, 'utf-8'))
+      gameSettings = settings[this.appName] || ({} as GameSettings)
     }
 
-    return settingsToReturn
+    // set specific keys depending on the platform
+    if (isMac) {
+      defaultSettings.wineCrossoverBottle = wineCrossoverBottle
+    } else if (isLinux) {
+      defaultSettings.wineVersion = wineVersion
+      defaultSettings.winePrefix = winePrefix || `${userHome}/.wine`
+
+      // fix winePrefix if needed
+      if (gameSettings.winePrefix) {
```

Comment:
```suggestion
      if (gameSettings.winePrefix?.includes('~')) {
```
so maybe we can avoid doing replace always?